onnxruntime/core/providers/webnn/builders/op_map.h

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Intel Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#include <map>
+#include "core/common/common.h"
```

Comment:
@Honry not sure this include line of header file is right or not, which is to address the `onnx_to_webnn_data_type_map` & `supported_fallback_integer_data_types `